src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        Random random = new Random();
+        FigureSupplier figureSupplier = new FigureSupplier();
+        final int randomRange = 10;
```

Comment:
[here](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names) you can read how to make 10 as constant